spec/latest/index.bs

```diff
@@ -233,6 +233,30 @@ array as having type `uint64`.
 For a given [[#key_format]], all named binary arrays for that format shall have a
 corresponding name in `data_types`.
 
+The following strings shall be used to describe data types:
+: "uint8"
+:: unsigned 8-bit integer
+: "uint16"
+:: unsigned 16-bit integer
+: "uint32"
+:: unsigned 32-bit integer
+: "uint64"
+:: unsigned 64-bit integer
+: "int8"
+:: signed 8-bit integer
+: "int16"
+:: signed 16-bit integer
+: "int32"
+:: signed 32-bit integer
+: "int64"
+:: signed 64-bit integer
+: "float32" or "float"
+:: IEEE binary32 floating point number
+: "float64" or "double"
```

Comment:
Thanks for the PR!

I would keep the spec specific and only allow `"float32"` and `"float64"`.

Reply:
Agree with using `float64` rather than `double`

Reply:
Okay! These have been removed.